### ISSUE TITLE
BSD dual licensing Bio.Pathway

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#2243]
Sub-issue of #898, we have not year heard from the original author of ``Bio.Pathway``, Tarjei Mikkelsen (CVS username ``tarjei``) about dual licensing their code.

Their old MIT email address no longer works, but this is probably them: https://www.linkedin.com/in/tarjei-mikkelsen-b448265b

Possibly they are @tmikkelsen on GitHub (dormant account created in 2014).

**Comment by tmikkelsen** [comment]
Yes - I hereby give my explicit permission to switch to the BSD 3-clause
license.

**Comment by peterjc** [comment]
Thank you @tmikkelsen - I've opened #2248 to implement this.

**Comment by peterjc** [comment]
The only file left to do here is ``Bio/KEGG/Enzyme/__init__.py`` where the following commit is problematic in not having consent:

https://github.com/biopython/biopython/commit/1c72a63b35db70d11c628b83a0269d1a9c6443a4?w=1

Adding public data files to the list of files tested is arguably not copyrightable in itself, but while a small code change, coping with the change to the ``PATHWAY`` line was creative work.
